imodels/util/data_util.py

```diff
@@ -117,6 +119,10 @@ def get_clean_dataset(dataset_name: str, data_source: str = 'imodels', data_path
             X, y = sklearn.datasets.make_friedman2(n_samples=200)
         elif dataset_name == 'friedman3':
             X, y = sklearn.datasets.make_friedman3(n_samples=200)
+        elif dataset_name == "radchenko_james":
```

Comment:
It's nice to have these here, but if it helps you iterate faster feel free to just have some kind of `get_simulated_data` function in `imodels-experiments`